com.b2international.snomed.ecl.ui.feature/feature.xml

```diff
@@ -28,4 +28,11 @@
          version="0.0.0"
          unpack="false"/>
 
+   <plugin
+         id="com.b2international.snomed.ecl.ide"
```

Comment:
Would it make sense to add the IDE plugin to `ecl.feature` instead of to `ecl.ui.feature`?

Reply:
I moved it over, we will see which place is better 😃